### PR TITLE
Angle testing

### PR DESCRIPTION
**Pull request by Nun-z**
Does this meet your requirements?  I transferred the changes to the angle_testing branch.

### CODE REVIEW

File: libretro/libretro.c [hunk 1]
```diff
@@ -468,19 +468,22 @@ void copy_file(char * ininame, char * fileName)
...
 }
-
-static LONG CALLBACK VectoredExcepHandler( PEXCEPTION_POINTERS exInfo )
-{
-    if(exInfo->ExceptionRecord->ExceptionCode == DBG_PRINTEXCEPTION_C)
-        return EXCEPTION_CONTINUE_EXECUTION;
-    else
-        return EXCEPTION_CONTINUE_SEARCH;
-}
+#if defined(HAVE_ANGLE) && defined(__MINGW32__)
```
Comment (m4xw) [inline libretro/libretro.c:471]: \n before #if

**Review by m4xw — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -477,7 +490,10 @@ ifeq ($(LLE), 1)
    COREFLAGS += -DHAVE_LLE
 endif
 
-COREFLAGS += -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -DUSE_FILE32API -DM64P_PLUGIN_API -DM64P_CORE_PROTOTYPES -D_ENDUSER_RELEASE -DSINC_LOWER_QUALITY -DTXFILTER_LIB -D__VEC4_OPT -DMUPENPLUSAPI -DHAVE_ANGLE
+COREFLAGS += -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -DUSE_FILE32API -DM64P_PLUGIN_API -DM64P_CORE_PROTOTYPES -D_ENDUSER_RELEASE -DSINC_LOWER_QUALITY -DTXFILTER_LIB -D__VEC4_OPT -DMUPENPLUSAPI
+ifeq ($(HAVE_ANGLE),1)
```
Comment (m4xw) [inline Makefile:494]: \n before ifeq

**Review by m4xw — CHANGES_REQUESTED** [review]

File: mupen64plus-core/src/main/rom.c [hunk 3]
```diff
@@ -679,8 +679,8 @@ void romdatabase_open(void)
         
         lines = strtok(NULL, "\n");
     }
-
-
+    
+
```
[the remark this answers is not in the document]
Comment (Nun-z) [reply mupen64plus-core/src/main/rom.c]: I'm not 100% on what you want here.  Is it something like what I just committed?

File: libretro/libretro.c (hunk 1 above)
Comment (Nun-z) [reply libretro/libretro.c:472]: New line was added.

File: Makefile (hunk 2 above)
Comment (Nun-z) [reply Makefile:495]: New line was added.

**Comment by m4xw** [comment]
I will get back at this PR when i wrap up the upcoming mupen core update

**Comment by m4xw** [comment]
Looks fine for now. If i find some issue later, i will let you know.

**Comment by m4xw** [comment]
@Nun-z can u check my angle branch (not angle-testing)?
https://cdn.discordapp.com/attachments/876520593636335646/894907594123132938/angle_mupen64plus_next_libretro.dll

**Comment by Nun-z** [comment]
Sorry, something came up today, but I should have a chance to look at it
within 24 hours.

**Pull request merged**